tests/processes/BearRunningTest.py

```diff
@@ -280,6 +279,7 @@ def test_run(self):
                              LOG_LEVEL.DEBUG,
                              LOG_LEVEL.DEBUG,
                              LOG_LEVEL.WARNING]
+
```

Comment:
I don't get it you asked me not to remove the newline. Do you want me to remove the newline now because that is what I had initially done?